lib/process.js

```diff
@@ -131,93 +131,99 @@ var parse = function (record, data, config) {
 	});
 };
 
-module.exports = function (config, url, cb) {
-	var parsedUrl = URL.parse(url.url);
-	var client = parsedUrl.protocol === 'https:' ? https : http;
-	var httpOptions = {
-		hostname: parsedUrl.hostname,
-		port: parsedUrl.port || (parsedUrl.protocol === 'https:' ? 443 : 80),
-		path: parsedUrl.pathname || '/',
-		method: 'GET',
-		auth: config.http && config.http.auth,
-		headers: config.http && config.http.headers
-	};
-	
-	if (!httpOptions.hostname) {
-		return {
-			url: url,
-			ok: false,
-			error: 'No hostname found'
+module.exports = function (data, cb) {
```

Comment:
I changed the config and url params for a simple object beacause I needed to add a other param (index). WDYT @nitriques ?

Reply:
👍